app/views/standard/additionalMetadataNavigation.scala.html

```diff
@@ -96,7 +96,7 @@ <h2 class="app-task-list__section">Folder uploaded: @folderUploaded</h2>
                                 @if(totalPages - currentPage > 3) {
                                         <li class="govuk-pagination__item govuk-pagination__item--ellipses">&ctdot;</li>
                                 }
-                            @if(totalPages != 1) {
+                            @if(totalPages != 1 && totalPages != 0) {
```

Comment:
Can we add one check which will work for both conditions: `@if(totalPages > 1)`?

Reply:
Yes you're right, that is much simpler :+1: